packages/main/src/plugin/container-registry.ts

```diff
@@ -876,6 +875,18 @@ export class ContainerProviderRegistry {
     ];
   }
 
+  public getFirstRunningPodmanContainerProvider(): InternalContainerProvider {
```

Comment:
I think it should be using getFirstRunningConnection code except that it does not return .api at the end


my worries are that here for example you're not sorting provider so podman might not be first while if we call getFirstRunningConnection it would have been first

AFAIK the only difference is returning `.api` or not

Reply:
AFAIK podman is the only one that has a defined libpodApi and i was using this one to fine the running container provider.

Reply:
In this case i'm only interested in a podman one so if you are using docker it throws

Reply:
Ok I see but then it should contain some comments saying that the other method can't be used or then we name method with Podman to be sure that we check podmanApi and not podApi

Reply:
Ok I see, maybe some comments